src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,51 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+
+    @Test
+    @DisplayName("1,2을 ,로 split 했을 때 1과 2로 분리되는지 테스트")
+    void split1() {
+        String[] result = "1,2".split(",");
+        assertThat(result).contains("1", "2");
+        assertThat(result).hasSize(2);
+        assertThat(result).containsExactly("1", "2");
```

Comment:
```suggestion
	assertThat(result).hasSize(2)
			.containsExactly("1", "2");
```
`assertj` 를 활용하시면 메소드 체이닝을 통해 직관적인 검증도 가능하니 참고해주세요!!